Source/WebCore/platform/VideoPixelFormat.cpp

```diff
@@ -43,6 +43,8 @@ VideoPixelFormat convertVideoFramePixelFormat(uint32_t format)
     if (format == kCVPixelFormatType_32ARGB)
         return VideoPixelFormat::RGBA;
     ASSERT_NOT_REACHED();
+#else
+    UNUSED_PARAM(format);
```

Comment:
I fixed this in https://github.com/WebKit/WebKit/pull/5202 but your change is byte-identical to mine, so instead of a conflict it should hopefully just disappear when you rebase.

Reply:
Had to update the commit message as well to reflect it.